variables.sh

```diff
@@ -18,3 +17,4 @@ NODE_VERSION=${NODE_VERSION:-14}
 NODE_GRPC_TOOLS_NODE_PROTOC_TS_VERSION=${NODE_GRPC_TOOLS_NODE_PROTOC_TS_VERSION:-5.1.3}
 NODE_GRPC_TOOLS_VERSION=${NODE_GRPC_TOOLS_VERSION:-1.11.1}
 NODE_PROTOC_GET_GRPC_WEB_VERSION=${NODE_PROTOC_GET_GRPC_WEB_VERSION:-1.2.1}
+BUILD_VERSION="${BUILD_VERSION:-${GRPC_VERSION}_0}"
```

Comment:
what's the `_0` for?

Reply:
This is to match the tag convention.
This var would either take the external value which will be set by the CI (and the tag), e.g tag `v1.37_0` will have `BUILD_VERSION=1.37_0`

or if built locally, it will generate a tag based on the grpc version with the added 0 (so again, `BUILD_VERSION=1.37_0`)

Reply:
now that I think about it, if it's going to be confusing, I can set it to generate locally as 
`BUILD_VERSION=1.37_local` because you might be generating locally `BUILD_VERSION=1.37_0` while actually wanting to generate tag `v1.37_1`, `v1.37_2` etc